src/LexicalTools/LexSenseLayouter.cs

```diff
@@ -112,23 +131,32 @@ private void AddExampleSentenceGhost(LexSense sense, int insertAtRow)
 		public int AddGhost(PalasoDataObject parent, IList<LexSense> list, bool isHeading, int insertAtRow)
 		{
 			string label = GetLabelForMeaning(list.Count);
-#if GlossMeaning
-			return MakeGhostWidget<LexSense>(list, insertAtRow, Field.FieldNames.SenseGloss.ToString(), label, "Gloss", isHeading, insertAtRow);
-#else
-			return MakeGhostWidget<LexSense>(parent,
-											list,
-											LexSense.WellKnownProperties.Definition,
-											label,
-											"Definition",
-											isHeading,
-											insertAtRow);
-#endif
+			if (GlossMeaningField)
+			{
+				return MakeGhostWidget<LexSense>(parent,
+												list,
```

Comment:
I wouldn't try to line up these parameters, because of the use of tabs. I know that it was already that way, but might as well fix it.